wiki/People/The_Team/Project_Loved_Team/id.md

```diff
@@ -1,46 +1,44 @@
 ---
-outdated: true
-outdated_since: 700437a67e7944ce70e934e9e7f2e451ef0b947a
 tags:
   - captain
   - captains
 ---
 
 # Project Loved Team
 
-**Project Loved Team** adalah anggota dari komunitas osu! yang mengelola [Project Loved](/wiki/Project_Loved), platform pemungutan suara untuk mempromosikan [beatmaps](/wiki/Beatmap) masuk ke kategori [Loved](/wiki/Beatmap/Category#loved).
+**Project Loved Team** merupakan gugus pengguna (*usergroup*) yang bertugas untuk mengelola keberlangsungan [Project Loved](/wiki/Project_Loved). Adapun Project Loved sendiri merupakan suatu sistem yang bertujuan untuk mempromosikan [beatmap-beatmap](/wiki/Beatmap) ke dalam kategori [Loved](/wiki/Beatmap/Category#loved) berdasarkan hasil jajak pendapat publik.
 
 ## Peran dan Tanggung Jawab
 
-Project Loved Team kebanyakan terdiri dari *kapten* untuk masing-masing [mode permainan](/wiki/Game_mode). Kapten bertanggung jawab untuk menominasikan beatmap ke kategori Loved dan bertindak sebagai perwakilan dari komunitas untuk mengetahui beatmap mana yang harus dipilih. Mereka juga menulis deskripsi singkat tentang beatmap yang dinominasi untuk membantu pembaca dapat merasakan setiap beatmap jika sebelumnya mereka belum pernah melihat atau memainkanya.
+Sebagian besar anggota Project Loved Team merupakan tokoh-tokoh komunitas terpilih yang bertindak selaku *kapten* (*captain*) pada [mode permainannya masing-masing](/wiki/Game_mode). Tiap-tiap kapten memiliki tanggung jawab untuk menominasikan beatmap-beatmap yang dianggap mewakili aspirasi komunitas secara luas ke dalam kategori Loved. Mereka juga menulis deskripsi singkat tentang beatmap-beatmap yang mereka nominasikan di setiap siklusnya.
```

Comment:
```suggestion
Sebagian besar anggota Project Loved Team merupakan tokoh-tokoh komunitas terpilih yang bertindak selaku *kapten* (*captain*) di masing-masing [mode permainan](/wiki/Game_mode). Tiap-tiap kapten memiliki tanggung jawab untuk menominasikan beatmap-beatmap yang dianggap mewakili aspirasi komunitas secara luas ke dalam kategori Loved. Mereka juga menulis deskripsi singkat tentang beatmap-beatmap yang mereka nominasikan di setiap siklusnya.
```
Menurutku bagusnya yang di hyperlink itu cuma yang `mode permainan` aja, sama kaya di en.md. Untuk penempatan kata `masing-masing`, kalau tanpa imbuhan `nya` diakhir kata `mode permainan`, pindah ke depan aja jadinya `masing-masing mode permainan`

Reply:
ah, okok

i give this some more thought and at last decided to go with ``[mode permainannya](/wiki/Game_mode) masing-masing`` instead